client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -63,6 +65,23 @@ function ReaderSubscriptionListItem( {
 		return <ReaderSubscriptionListItemPlaceholder />;
 	}
 
+	function recordEvent( name ) {
```

Comment:
is there any way to avoid creating all the `recordX` functions within the component? would they trigger unnecessary rerenders?